tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -1037,3 +1455,1106 @@ def test_checkpoint_run_adds_validation_ids_to_expectation_suite_validation_resu
+        data_context=context,
+        config_version=1,
+        run_name_template="%Y-%M-foo-bar-template",
+        expectation_suite_name="my_expectation_suite",
+        action_list=common_action_list,
+    )
+    with pytest.raises(
+        gx_exceptions.DataContextError, match=r"expectation_suite .* not found"
+    ):
+        checkpoint.run(validations=[{"batch_request": batch_request}])
+
+    assert len(context.validations_store.list_keys()) == 0
+
+    context.add_expectation_suite("my_expectation_suite")
+    result = checkpoint.run(validations=[{"batch_request": batch_request}])
+
+    assert len(context.validations_store.list_keys()) == 1
+    assert result["success"]
+
+
+@pytest.mark.integration
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
+@pytest.mark.slow  # 1.31s
+def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_run_batch_request_multi_validation(
+    mock_emit,
+    titanic_data_context_with_fluent_pandas_and_spark_datasources_with_checkpoints_v1_with_empty_store_stats_enabled,
+    common_action_list,
+    batch_request_as_dict,
+    failed_multi_validation_checkpoint_run_usage_statistics_expected_events,
+    successful_multi_validation_checkpoint_run_usage_statistics_expected_events,
+):
+    context: FileDataContext = titanic_data_context_with_fluent_pandas_and_spark_datasources_with_checkpoints_v1_with_empty_store_stats_enabled
+
+    batch_request_0: dict = copy.deepcopy(batch_request_as_dict)
+
+    batch_request_1: dict = {
+        "datasource_name": "my_pandas_dataframes_datasource",
+        "data_asset_name": "my_dataframe_asset",
+    }
+
+    batch_request_2: dict = {
+        "datasource_name": "my_spark_filesystem_datasource",
+        "data_asset_name": "users",
+    }
+
+    batch_request_3: dict = {
+        "datasource_name": "my_spark_dataframes_datasource",
+        "data_asset_name": "my_dataframe_asset",
+    }
+
+    checkpoint: Checkpoint = Checkpoint(
+        name="my_checkpoint",
+        data_context=context,
+        config_version=1,
+        run_name_template="%Y-%M-foo-bar-template",
+        expectation_suite_name="my_expectation_suite",
+        action_list=common_action_list,
+    )
+    with pytest.raises(
+        gx_exceptions.DataContextError, match=r"expectation_suite .* not found"
+    ):
+        # noinspection PyUnusedLocal
+        result = checkpoint.run(
+            validations=[
+                {"batch_request": batch_request_0},
+                {"batch_request": batch_request_1},
+                {"batch_request": batch_request_2},
+                {"batch_request": batch_request_3},
+            ]
+        )
+
+    assert mock_emit.call_count == 1
+
+    expected_events = (
+        failed_multi_validation_checkpoint_run_usage_statistics_expected_events
+    )
+    # noinspection PyUnresolvedReferences
+    actual_events: List[unittest.mock._Call] = mock_emit.call_args_list
```

Comment:
asserting the list of events is sufficient I think, we don't need to assert the whole test body here

Reply:
🤔

Reply:
@donaldheppner My logic for keeping the full event body is due to the fact that the `BatchRequest` composition has changed, and we must make sure that the anonymized events reflect that.  Having said that, I would like to remove these tests for now, because Usage Statistics will not work until fully integrated with the JSON schema and the Usage Statistics service; we should address it separately (I will create a ticket).  Thank you.